packages/config/src/VitessceAutoConfig.js

```diff
@@ -381,7 +543,7 @@ async function generateConfig(url, vc) {
     }
     // this piece of code can be removed once these props are added by default to layerController
     // see this issue: https://github.com/vitessce/vitessce/issues/1454
-    if (v[0] === 'layerController' && configInstance instanceof OmeTiffAutoConfig) {
+    if (v[0] === 'layerController') {
       view.setProps({
         disable3d: [],
         disableChannelsIfRgbDetected: true,
```

Comment:
this is now added for every layerController, no matter of the file type. Should I revert this change?